lightning-net-tokio/src/lib.rs

```diff
@@ -634,8 +639,53 @@ mod tests {
 	async fn basic_threaded_connection_test() {
 		do_basic_connection_test().await;
 	}
+
 	#[tokio::test]
 	async fn basic_unthreaded_connection_test() {
 		do_basic_connection_test().await;
 	}
+
+	async fn race_disconnect_accept() {
+		// Previously, if we handed an already-disconnected socket to `setup_inbound` we'd panic.
+		// This attempts to find other similar races in by opening connections and shutting them
```

Comment:
nit: can remove "in" (unless a word was missed after it)
```suggestion
		// This attempts to find other similar races by opening connections and shutting them
```